scripts/testnet.sh

```diff
@@ -156,3 +156,11 @@ add_keys() {
     echo "Authorized keys file not found"
   fi
 }
+
+disable_hardhat_auto_mining() {
```

Comment:
This script file will be removed eventually. Please add this function inline for now in the integration test script.

Reply:
changed